internal/service/autoscaling/groups_data_source_test.go

```diff
@@ -24,11 +24,20 @@ func TestAccAutoScalingGroupsDataSource_basic(t *testing.T) {
 				Config: testAccCheckGroupsConfig(sdkacctest.RandInt(), sdkacctest.RandInt(), sdkacctest.RandInt()),
 			},
 			{
-				Config: testAccCheckGroupsWithDataSourceConfig(sdkacctest.RandInt(), sdkacctest.RandInt(), sdkacctest.RandInt()),
```

Comment:
The resources were not being created with this function, but rather using the `testAccCheckGroupsConfig` above resources. Therefore, I removed some of this cruft.